### ISSUE TITLE
Transaction amount field 

### ISSUE THREAD

**Issue by kngys** [AY2425S1-CS2103T-F13-4/tp#131]
Just wondering whether it would make more sense to store the amount field as a double instead of an int? Since in the real world it's possible to have amounts that are not integers.

**Comment by darkmoongreatsword** [comment]
Makes sense. I'll adjust it tgt with the parser do we want to limit the number of decimal points to 2?

**Comment by kngys** [comment]
I think 2 decimals places should be fine

**Comment by reginateh** [comment]
Duplicate of issue #105

**Comment by kngys** [comment]
Has this been resolved yet? Would be better to complete this soon in case it leads to changes in other classes or UI.
